.github/workflows/main.yml

```diff
@@ -677,12 +677,6 @@ jobs:
       env:
         RUST_BACKTRACE: 1
 
-    # Crude check for whether
-    # `unsafe_disable_continuation_linearity_check` makes the test
-    # `cont_twice` fail.
-    - run: |
-        (cargo test --features=unsafe_disable_continuation_linearity_check --test wast -- --exact Cranelift/tests/misc_testsuite/typed-continuations/cont_twice.wast && test $? -eq 101) || test $? -eq 101
-
```

Comment:
Why are you deleting this test? It should still fail.

Reply:
See comment on wast.rs